src/components/cc-tile-deployments/cc-tile-deployments.js

```diff
@@ -36,6 +37,10 @@ export class CcTileDeployments extends LitElement {
     this.state = { type: 'loading' };
   }
 
+  /**
+   * @param {DeploymentTileInfo['state']} state
```

Comment:
question: just curious, would `DeploymentTileInfo.state` be valid or you must use the bracket syntax?

Reply:
The dot notation does not work for types but I didn't know why so your question forced me to search for an answer: https://www.totaltypescript.com/why-you-cant-use-dot-notation-on-types